api/build.gradle

```diff
@@ -309,30 +306,6 @@ dependencies {
   generatedCompile 'com.fasterxml.jackson.datatype:jackson-datatype-joda:2.9.6'
 }
 
-swaggerSources {
```

Comment:
Could you verify that validateSwagger still validates swagger files after this change? I'm not 100% sure, but I vaguely recall setting up some non-obvious hand-off to the swagger plugin to feed into that validation.

Reply:
Thank you, you're correct. This is required. The validation code runs, but doesn't see the files so doesn't fail. This is very non-obvious - I'll close this and look for a better way to make it clear that these are used.

Reply:
Sorry, I probably should have left a big comment here explaining why we had this.